pep/app/pods/most-viewed/controller.ts

```diff
@@ -123,6 +123,7 @@ export default class MostViewed extends Controller {
             filterRootKey: null,
             filterList: ['author', 'title', 'sourcename', 'period', 'queryType']
         });
+        delete queryParams.limit;
```

Comment:
we prob should eventually update that addon's util to allow `limit: null` to do this.

Reply:
I can do that later on if your ok with that